src/dyn_subgrid/dynSubgridControlMod.F90

```diff
@@ -305,8 +317,19 @@ subroutine check_namelist_consistency
           write(iulog,*) 'ERROR: do_harvest can only be true if either use_cn or use_fates are true'
           call endrun(msg=errMsg(sourcefile, __LINE__))
        end if
-    end if    
-    
+    end if
+
+    if (dyn_subgrid_control_inst%do_grossunrep) then
+       if (.not. use_cn) then
+          write(iulog,*) 'ERROR: do_grossunrep can only be true if use_cn is true'
```

Comment:
Technically when running with FATES it will trigger this first error, rather than the second. To get the right error printed the order of the if statements should be reversed. And you could add a note about it.

I don't know this is worth doing though since the testing is done though.

Reply:
Sam went ahead with this change so this is all good.